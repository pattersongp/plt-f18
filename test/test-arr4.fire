func int main = () => {
        
        array[int, str] a;
        a[0] = "GATE";
        a[1] = "GATE";
        a[2] = "PARASAMGATE";
        a[3] = "BODHI SAVHA";

        int j = 0;
        while(j < 4) {
        	sprint(a[j]);
        	j = j + 1;
        }

        return 0;
}