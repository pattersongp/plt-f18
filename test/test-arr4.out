GATE
GATE
PARASAMGATE
BODHI SAVHA